fdb-record-layer-core-shaded/fdb-record-layer-core-shaded.gradle

```diff
@@ -23,7 +23,7 @@ apply plugin: 'com.github.johnrengelman.shadow'
 def coreProject = ":${ext.coreProjectName}"
 
 dependencies {
-  compile project(coreProject)
+  implementation project(coreProject)
```

Comment:
Is this right for shaded artifacts (I don't actually know)? The relationship here is a little weird, in that a user who wants to use the shaded artifacts should take the `fdb-record-layer-core-shaded` artifact _instead_ of `fdb-record-layer-core`, so maybe `implementation` is correct?

Reply:
I think that was the reasoning. `implementation` get the shader access to the bits, which it then re-publishes, rather than passing through.

Reply:
left as-is